config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  devise_for :users
```

Comment:
- [ ] an unauthorized user should be redirected to the sign_up or sign_in page, not the user index page